api/envoy/api/v2/core/config_source.proto

```diff
@@ -180,3 +180,14 @@ message ConfigSource {
   // turn expect to be delivered.
   ApiVersion resource_api_version = 6 [(validate.rules).enum = {defined_only: true}];
 }
+
+// [#not-implemented-hide:]
+// Configuration of the event reporting service endpoint.
+message EventServiceConfig {
```

Comment:
Arguably could be in its own `event_service_config.proto` but I don't feel that strongly (it's easy enough to change later with breaking things).

Reply:
sure, why not:)